cppcache/acceptance-test/SNITest.cpp

```diff
@@ -52,6 +52,8 @@ class SNITest : public ::testing::Test {
   ~SNITest() override = default;
 
   void SetUp() override {
+    TearDown();
```

Comment:
same thing here... `gtest` uses `TearDown` as a special method... I wonder if it would be better to have a utility method for both that is called from the constructor in C# and `SetUp` in this test...

Reply:
Yep

Reply:
Agreed. Employing a utility function as above.

Reply:
This looks like it was merged with `TearDown()` being called instead of `cleanupDocker()` function...